packages/kbn-optimizer/limits.yml

```
@@ -27,7 +27,7 @@ pageLoadAssetSize:
   indexLifecycleManagement: 107090
   indexManagement: 140608
   infra: 184320
-  fleet: 95000
+  fleet: 110025
```

```suggestion
  fleet: 100000
```
Mind if we do this instead?  It's the limit we try to keep most teams under.